lib/moonbeam-xcm-transactor/XcmTransactorV2.sol

```diff
@@ -0,0 +1,154 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity ^0.7.6;
+pragma abicoder v2;
+
+// The XcmTransactorV2 contract's address.
+address constant XCM_TRANSACTOR_V2_ADDRESS = 0x000000000000000000000000000000000000080D;
```

Comment:
Where does this come from? 

Will this need to be updated sometime in the future or is this static?

Reply:
So this whole file comes from the Moonbeam repo. To add this as a dependency we would need to add the whole repo as a git submodule which is too much overhead just to get this file so we copied it over.

Reply:
Maybe we should add a note where we can look up, if this changes? 

I guess this is their hardcore contract on the runtime side

Reply:
This shouldn't change tho. The contract is deployed, it has a constant address (line 6 we are looking at) and this file is just an interface to interact with that contract. I can add a comment linking it to the original repo

Reply:
I guess it is a precompile rather than a contract. Hence, the address could change. But most likely won't

Reply:
They would most likely release v3 for that purpose, the precompile/contract is deployed now for v1 and V2 so I don't think we have anything to be concerned here tbh